src/main/java/pl/javastart/task/Microphone.java

```diff
@@ -0,0 +1,24 @@
+package pl.javastart.task;
+
+public class Microphone {
+    String producer;
+    String model;
+    double price;
+    String transducerType;
+    double minSampleRate;
+    double maxSampleRate;
+    int bitRate;
+    boolean isWireless;
+
+    Microphone(String prod, String mod, double pr, String typeOfTransducer, double minSample, double maxSample, int bRate,
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu